pkgs/applications/office/qnotero/default.nix

```
@@ -26,6 +26,13 @@ python3Packages.buildPythonPackage rec {
     wrapQtApp "$out"/bin/qnotero
   '';
 
+  postInstall = ''
+    mkdir $out/share
+    mv $out/usr/share/applications $out/share/applications
+    substituteInPlace $out/share/applications/qnotero.desktop \
+      --replace "Icon=" "Icon=$out"
```

It would be nice if the icon can just be set to the icon name and the icon be linked into the default icon Dir so that it can be changed by themes.
```suggestion
      --replace "Icon=/usr/share/qnotero/resources/light/qnotero.png" "Icon=qnotero"
    
    mkdir -p $out/share/icons/hicolor/64x64/apps
    ln -s $out/usr/share/qnotero/resources/light/qnotero.png \
      $out/share/icons/hicolor/64x64/apps/qnotero.png
```

---

Agreed! I am not too familiar with that API though, I will commit and squash